onnxruntime/wasm/js_internal_api.js

```diff
@@ -160,6 +160,10 @@ Module['jsepInit'] = (backend, alloc, free, copy, copyAsync, createKernel, relea
   };
 
   // replace the original functions with asyncified versions
+  Module['_OrtCreateSession'] = runAsync(jsepWrapAsync(
```

Comment:
_OrtCreateSession is not a "Run" entry. Use `jsepWrapAsync` only. ( see _OrtBindInput )

Reply:
@fs-eire, thanks, I fixed that but still got same error.

I tried to add some console logs to `jsepWrapAsync` to see if it is actually called by WebNN EP, the result is 'No'. I can see these console logs with webgpu backend. Looks like we miss some other config to let webnn ep use async functions in this js_internal_api.js file.

Reply:
I think I know where it is missing.

You need to run `await initJsep(getInstance(), env, adapter);` in wasm-core-impl.ts. However, currently this initialization only called with webgpu. If you want to test, you can try move that line out of the if clause. I will think how to do this in a better way.

Reply:
You can change that condition on line 87 from
```ts
if (!BUILD_DEFS.DISABLE_WEBGPU && epName === 'webgpu') {
```
to
```ts
if (!BUILD_DEFS.DISABLE_WEBGPU && (epName === 'webgpu' || epName === 'webnn')) {
```

Reply:
Awesome, that works! Thanks you so much @fs-eire! I will continue to update the WebNN `compute` part.

<img width="497" alt="image" src="https://github.com/microsoft/onnxruntime/assets/3271201/69addf58-5a22-4f28-97b2-a1f1fb5b8260">